tardis/energy_input/indivisible_packets.py

```diff
@@ -615,7 +615,7 @@ def process_packet_path(packet):
     return packet, ejecta_energy_gained
 
 
-@njit(parallel=True)
+@njit
```

Comment:
Please set this to use the `njit_dict_no_parallel` from montecarlo/montecarlo_numba.  This does raise a good point that our numba config shouldn't be in the numba folder, but instead global.